app/src/experiments.py

```diff
@@ -85,7 +86,10 @@ def get_question(example: dict) -> str:
 def query_pipeline(example: dict) -> TaskOutput:
     question = get_question(example)
     logger.info("Getting answer for: %r", question)
-    response = create_pipeline().run_api(query=question)
+    kwargs = {"query": question}
+    if prompt_version:
+        kwargs["prompt_version_id"] = prompt_version  # type: ignore[unreachable]
+    response = create_pipeline().run_api(**kwargs)
```

Comment:
Above lines 89-92 could be simplified to:
```suggestion
    response = create_pipeline().run_api(query=question, prompt_version_id=prompt_version or "")
```